themes/bootstrap3/templates/RecordTab/usercomments.phtml

```diff
@@ -6,7 +6,7 @@
 <div class="comment-list">
   <?=$this->render('record/comments-list.phtml')?>
 </div>
-<form class="comment-form" name="commentRecord" action="<?=$this->recordLink()->getActionUrl($this->driver, 'AddComment')?>" method="post">
+<form class="comment-form" name="commentRecord" action="<?=$this->recordLink()->getActionUrl($this->driver, 'AddComment')?>" method="post" data-lightbox data-lightbox-onclose="addRecordCommentCallback">
```

Comment:
data-lightbox-onsubmit cannot be used to suppress the lightbox popup... I don't know if we have a mechanism for that.